__docs__/wonder-blocks-tooltip/tooltip.stories.tsx

```diff
@@ -509,3 +512,53 @@ export const InTopCorner = {
         </View>
     ),
 };
+
+export const InCorners = {
```

Comment:
**suggestion:** Same for including a description to this history here. To expand on that, probably worth mentioning that we provide an spacing by default, but that it can be overwitten using .... if needed.